app/assets/javascripts/base.js.erb

```diff
@@ -4,9 +4,11 @@
 //= require initializePage
 //= require utilities/getImageForLink
 //= require @honeybadger-io/js/dist/browser/honeybadger.js
+//= require @hotwired/turbo
 //= require i18n
 //= require ahoy.js/dist/ahoy.js
 
+Turbo.session.drive = false
```

Comment:
This was an unexpected turn of events. I don't know if this is worth sharing if this fixes the problems you saw.

Reply:
right! I think it would be worth sharing out for sure.